assets/common/src/mlflow_model_local_validation/run_model_sanity_validations.py

```diff
@@ -81,10 +81,10 @@ def get_dict_from_comma_separated_str(dict_str: str, item_sep: str, kv_sep: str)
     item_sep = item_sep.strip()
     kv_sep = kv_sep.strip()
     if len(item_sep) > 1 or len(kv_sep) > 1:
-        tc_exception("Provide single char as separator")
+        tc_log("Provide single char as separator")
```

Comment:
We seem to be using this method at some places for logging and at other places in instance of a logger. Is there a reason for that?

Reply:
Wondering if we could use logger.info everywhere or something similar.

Reply:
will be addressing in a separate PR. This is just to fix a bug that went in case of exception